test/ruby/test_string.rb

```diff
@@ -513,6 +513,11 @@ def test_concat
     assert_raise(RuntimeError) { 'foo'.freeze.concat('bar') }
   end
 
+  def test_concat_literals
+    s="."*50; "#{s}x".encoding
```

Comment:
The test repeats ``"#{s}x".encoding``, which I believe to be unnecessary.
I would also add spaces for readability, so ``s="."*50; "#{s}x".encoding`` would become ``s = "." * 50``.

Reply:
Thanks for your comment! 😄
I fixed it! https://github.com/ruby/ruby/pull/1626/commits/abc4f2be2682bb206a7a74b02e32aab35dae0ba2